cluster/examples/kubernetes/ceph/upgrade-from-v0.8-replace.yaml

```diff
@@ -41,7 +41,7 @@ rules:
   - update
   - delete
 - apiGroups:
-  - extensions
+  - apps
```

Comment:
This is only used for upgrades from 0.8 to 0.9, so we'll need to revert this. We'll need something similar for the v0.9 to v1.0 upgrade before we release though